server/controllers/finance/reports/financial.patient.handlebars

```diff
@@ -40,12 +40,12 @@
         {{/each}}
       </tbody>
       <tfoot>
-        <tr>
+        <tr style="background-color: #ededed;">
```

Comment:
In the future, it would be better to use a descriptive class name.  This is not a huge problem though.